src/nuklear_style.c

```diff
@@ -490,8 +490,9 @@ nk_style_from_table(struct nk_context *ctx, const struct nk_color *table)
     tab->background         = nk_style_item_color(table[NK_COLOR_TAB_HEADER]);
     tab->border_color       = table[NK_COLOR_BORDER];
     tab->text               = table[NK_COLOR_TEXT];
-    tab->sym_minimize       = NK_SYMBOL_TRIANGLE_RIGHT;
-    tab->sym_maximize       = NK_SYMBOL_TRIANGLE_DOWN;
+    tab->sym_minimize       = NK_SYMBOL_PLUS;
+    tab->sym_maximize       = NK_SYMBOL_MINUS;
+    tab->sym_child          = NK_SYMBOL_NONE;
```

Comment:
I think this is a breaking change visually for people who use the default style

Reply:
Generally I'm indifferent on this one. A quick google search seems to suggest that triangles are a tiny bit more modern than `+` `-` though. So I think this can be left out of this big PR.